services/settings.py

```diff
@@ -0,0 +1,8 @@
+import os
+
+from django.utils import importlib
+
+
+# get the right settings module
+settings = importlib.import_module(
+    os.environ.get('DJANGO_SETTINGS_MODULE', 'settings_local'))
```

Comment:
😱 😨 

You removed `zamboni` and `freddo` words from this part of our code 🔥

Reply:
I don't even know what freddo was!

Reply:
I think Andy told me during my onboarding but I'm sure there's a reason I forgot